docs/warehouse/warehouse_starter_kit.md

```diff
@@ -15,7 +15,7 @@
 
 ## Trips
 On a given day:
-* [fct_daily_scheduled_trips](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_scheduled_trips)
+* [fct_scheduled_trips](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_scheduled_trips)
```

Comment:
Ope sorry just one thing -- may want to update the link body here too to: https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_scheduled_trips